backend/src/main/java/com/htecgroup/skynest/controller/UserController.java

```diff
@@ -264,14 +269,17 @@ public ResponseEntity<String> confirmPasswordReset(
   }
 
   @Operation(summary = "Get User with that id")
+  @PreAuthorize("hasAuthority(T(com.htecgroup.skynest.model.entity.RoleEntity).ROLE_WORKER)")
```

Comment:
What if you need to extract this endpoint to be acceptable with additional role? Check @RolesAllowed

Reply:
We're only using authorities right now, so users don't have roles yet. Not sure how to do that.

Reply:
From the name that you use you already have roles :) so try with annotation that I sent.. here is a link with explanation https://stackoverflow.com/questions/43961625/rolesallowed-vs-preauthorize-vs-secured. From that link you will see what needs to be removed in the new configuration class